sortcom.cpp

```diff
@@ -0,0 +1,57 @@
+//sorts array of strings with user written comparison function

```

Comment:
can you move the file inside sorting/sort_strings

Reply:
how to move it please tell me its my first contribution

Reply:
Sure, you need to just copy the file to sorting/sort_strings directory and then delete this file.

finally 
```git pull -r origin master
git add .
git commit -m "moved files"
git push origin master```